src/components/createuser.js

```diff
@@ -22,25 +23,21 @@ class createuser extends Component {
 
   handleCreation = async () => {
     try {
-      console.log(this.state);
-      // const body = {};
-
-      // axios.post("/create_user", body).then(response => {
-      //   this.setState({ created: true });
-      //   this.props.history.push("/app/home_page");
-      // });
+      const { firstName, lastName, gender, email, password, username } = this.state;
```

Comment:
@DrewStableBoi , it looks like you're not sending the phone number. Is that on purpose? I think we should be sending it

Reply:
@DrewStableBoi , also, is there a reason you are not using this.setstate?